apps/lockfile-explorer/src/cli/explorer/ExplorerCommandLineParser.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { ConsoleTerminalProvider, type ITerminal, Terminal } from '@rushstack/terminal';
+import {
+  type CommandLineFlagParameter,
+  CommandLineParser,
+  type CommandLineStringParameter
+} from '@rushstack/ts-command-line';
+
+import { StartAction } from './actions/StartAction';
+
+const EXPLORER_TOOL_FILENAME: 'lockfile-explorer' = 'lockfile-explorer';
+
+export class ExplorerCommandLineParser extends CommandLineParser {
+  public readonly globalTerminal: ITerminal;
+  private readonly _terminalProvider: ConsoleTerminalProvider;
+  private readonly _debugParameter: CommandLineFlagParameter;
+  private readonly _subspaceParameter: CommandLineStringParameter;
+
+  public constructor() {
+    super({
+      toolFilename: EXPLORER_TOOL_FILENAME,
+      toolDescription: 'lockfile-lint is a tool for linting lockfiles.'
+    });
+
+    this._debugParameter = this.defineFlagParameter({
+      parameterLongName: '--debug',
+      parameterShortName: '-d',
+      description: 'Show the full call stack if an error occurs while executing the tool'
+    });
+
+    this._subspaceParameter = this.defineStringParameter({
+      parameterLongName: '--subspace',
+      argumentName: 'SUBSPACE_NAME',
+      description: 'Specifies an individual Rush subspace to check.'
+    });
```

Comment:
```suggestion
    this._subspaceParameter = this.defineStringParameter({
      parameterLongName: '--subspace',
      argumentName: 'SUBSPACE_NAME',
      description: 'Specifies an individual Rush subspace to check.',
      defaultValue: 'default'
    });
```

And then you should be able to change the type of the parameter to `IRequiredCommandLineStringParameter`